server/src/controllers/sections/getProjectSectionsController.ts

```diff
@@ -0,0 +1,19 @@
+import { NextFunction, Request, Response } from 'express';
+import { getProjectSectionsQuery } from '../../database';
+
+const getProjectSectionsController = async (req: Request, res: Response, next: NextFunction) => {
+  const { id } = req.params;
+  try {
+    // Get project sections
+    const { rows } = await getProjectSectionsQuery(+id);
```

Comment:
you should check if there is no section by rowCount

Reply:
Yup, I will do it.
Thanks Amran